### PR TITLE
github: add PR labeling action

### PR DESCRIPTION
**Pull request by NicolasT**
See: https://github.com/actions/labeler

### CODE REVIEW

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Labeler
+on: [pull_request]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
```
Comment (slaperche-scality) [inline .github/workflows/label.yml:7]: Why the empty lines around this one?

Comment (NicolasT) [reply .github/workflows/label.yml:7]: https://github.com/actions/starter-workflows/blob/master/automation/label.yml

Comment (slaperche-scality) [reply .github/workflows/label.yml:7]: Ho ok, fair enough.

I simply looked at the the README from your link (https://github.com/actions/labeler) where there were no spaces around this line.

**Review by slaperche-scality — APPROVED** [review]

**Pull request merged**